### ISSUE TITLE
decomplect data processing

### ISSUE THREAD

**Issue by ajschumacher** [DCActionforChildren/dcaction#126]
It scares me that the data processing relies on a Google spreadsheet: https://docs.google.com/spreadsheet/ccc?key=0AliQRBwTvad9dHpVQkltc05MS2FzdGpCRFZSb0djdFE#gid=35 so to continue my opining, I think it would be great to remove that from the flow and perhaps even wrap the process in something like drake https://github.com/Factual/drake (or just a simple shell script, whatever)

I do see that there are a bunch of comments in the Google spreadsheet, which makes it seem like there's may be a connection with #125 as well.

**Comment by mjrich** [comment]
Agreed that there are much better alternatives out there for safely processing the data. The original idea was some interface that DCAC could directly and comfortably work with, hence spreadsheets.  Open to other workflow suggestions, but would want something that is easily debuggable by DCAC or human readable by any member of the general public to aid in transparency.  perhaps we could go with a set of CSV files that are created as close as possible to the direct sources, and then stiched together in a final step to create the neighborhoods.csv.  This could be a pretty well-scoped project for a small team of volunteers.

**Comment by ajschumacher** [comment]
Quite agree - I don't know what the ultimate awesome solution is yet, but I'd like to!

**Comment by ajschumacher** [comment]
I had this in #124, but it makes more sense here:

good ideas from @jotok, talking about having to be careful about getting variable/column names right in the spreadsheet:

> Is this something we could simplify by having the script output the same
> variable names that the front-end code is expecting? It would be an easy
> change!
> 
> While we're documenting the process, we should keep an eye out for other
> steps that could be automated. Less time fussing with spreadsheets means
> more time for cool visualizations and policy proposals.
